libcxx/utils/ci/run-buildbot

```diff
@@ -217,7 +217,7 @@ function test-armv7m-picolibc() {
         "${@}"
 
     ${NINJA} -vC "${BUILD_DIR}/compiler-rt" install
-    mv "${BUILD_DIR}/install/lib/armv7m-none-unknown-eabi"/* "${BUILD_DIR}/install/lib"
+    mv "${BUILD_DIR}/install/lib/armv7m-unknown-none-eabi"/* "${BUILD_DIR}/install/lib"
```

Comment:
This triple is 'used' by the version of the clang on the builder. Please remove this update, I'm in touch with @DavidSpickett who will make sure it gets updated at the right time.

Reply:
https://github.com/llvm/llvm-project/pull/90722

You can remove all changes to this file.